rs/bitcoin/kyt/src/state.rs

```diff
@@ -45,10 +46,48 @@ pub enum FetchTxStatus {
 /// input address will be computed and filled in.
 #[derive(Clone, Debug)]
 pub struct FetchedTx {
-    pub tx: Transaction,
+    pub tx: TransactionInputOutput,
```

Comment:
The name is a bit cumbersome and not entirely accurate because it only contains the output _addresses_. How about something like `TransactionKytData` because you capture all the KYT relevant data of the transaction? Other suggestions are welcome, of course.   :slightly_smiling_face: